doc/data-formats.md

```diff
@@ -105,6 +105,15 @@ These files are denoted with the prefix `pbta-fusion`.
 
 * `pbta-histologies.tsv`
 
+##### Mapping Between DNA-seq and RNA-seq Data for the Same Sample
+
+Many analyses rely on examining DNA-seq (e.g., WGS/WXS/Panel) and RNA-seq data together.
+The identifiers in the `Kids_First_Biospecimen_ID` column of the `pbta-histologies.tsv` file will be non-overlapping for different experimental strategies, as these identifiers map to a _library or assay_.
```

Comment:
Can we also add 1-2 bullets clarifying the specific definition/differences among `Kids_First_Biospecimen_ID` vs `sample_id` vs `Kids_First_Participant_ID`? 
My understanding has been `sample_id` reflects the individual, but are there multiple samples per individual? If so, let's clarify this aspect here too.

Reply:
That is somewhat covered here: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/doc/data-formats.md#independent-specimen-lists
I can link to the independent specimen list section here.

Reply:
I see it now - I had missed it before because columns aren't specifically there. The link you sent reads (my emphasis added), "For participants with multiple tumor specimens, independent specimen lists are provided as TSV files with **columns for participant ID and specimen ID (methods described here).**" Would be helpful to explicitly state that these columns are respectively `Kids_First_Participant_ID` and `Kids_First_Biospecimen_ID`.